src/SignalingClient.ts

```diff
@@ -57,9 +58,12 @@ interface WebSocketMessage {
  */
 export class SignalingClient extends EventEmitter {
     private static DEFAULT_CLIENT_ID = 'MASTER';
+    private static readonly WS_PING_PONG_INTERVAL_MS = 10 * 1000;
```

Comment:
Especially since customers are charged per message sent, this value should be configurable. Similarly, it should be configurable if this message is even sent at all.

Reply:
Similarly, a comment in the README about the ping/pong would be valuable too.